BACKEND/index.js

```diff
@@ -0,0 +1,132 @@
+const mongoose = require('mongoose');
+let canteenmenu=require("./model.js");
+const bodyParser=require("body-parser")
+const ejs = require("ejs");
+
+
+const express=require('express');
+const app=express();
+const path=require('path');
+const port=80;
+app.set('view engine', 'ejs');
+app.use(express.static("public"));
+
+// import canteenmenu from "./model.js";
+
+app.use(bodyParser.urlencoded({
+  extended: true
+}));
+
+main().catch(err => console.log(err));

```

Comment:
Dont use a separate line for catch block, connect like this:

`.connect(url)
    .then(() => {
      console.log("Connected to Database")
        app.listen(PORT);
    })
    .catch(err => { 
      console.log("Error in connecting to database")
        console.log(err)
    });`

Reply:
@Aryan3902 Actually, he is using the async await syntax for the main function. Thus, his original method was perfectly valid for dealing with errors in the async main() function.